notify-sharp must depend on correct dbus-sharp

Hi, 

when emerging sparkleshare (which in turn pulls in notify-sharp), I get this error when configuring notify-sharp-3.0.3:

```
configure: error: Package requirements (dbus-sharp-2.0 >= 0.7                               dbus-sharp-glib-2.0 >= 0.5) were not met:

No package 'dbus-sharp-2.0' found
No package 'dbus-sharp-glib-2.0' found
```

In fact dbus-sharp(-glib) are not pulled in. This is how my current dbus-sharp installation looks like:

```
janeway src # eix dbus-sharp
[U] dev-dotnet/dbus-sharp
     Available versions:  
     (1.0)  0.7.0-r2
     (0)    [m]~0.7.0-r2[1] [m]~0.8.1[1]
     (2.0)  0.8.1
       {debug developer}
     Installed versions:  0.7.0-r2(1.0)(05:30:17 PM 11/18/2015)
     Homepage:            https://github.com/mono/dbus-sharp
     Description:         D-Bus for .NET

[U] dev-dotnet/dbus-sharp-glib
     Available versions:  
     (1.0)  0.5.0-r1
     (0)    [m]~0.6[1]
     (2.0)  0.6.0
       {debug developer}
     Installed versions:  0.5.0-r1(1.0)(05:30:30 PM 11/18/2015)
     Homepage:            https://github.com/mono/dbus-sharp
     Description:         D-Bus for .NET: GLib integration module

[1] "dotnet" /var/lib/layman/dotnet
```

As far as I can tell, the requirements of the ebuild are satisfied since the ebuild only matches the version number, not the package slot. I have now manually installed dbus-sharp and dbus-shap-glib from the 2.0 slot (not from the dotnet overlay!), and notify-sharp builds fine.

--- a/dev-dotnet/notify-sharp/notify-sharp-3.0.3.ebuild
+++ b/dev-dotnet/notify-sharp/notify-sharp-3.0.3.ebuild
@@ -19,8 +19,8 @@ IUSE="doc"
 
 RDEPEND=">=dev-lang/mono-4.0.2.5
 	>=dev-dotnet/gtk-sharp-2.12.21:2
-	>=dev-dotnet/dbus-sharp-0.7:*
-	>=dev-dotnet/dbus-sharp-glib-0.5:*
+	>=dev-dotnet/dbus-sharp-0.7:2.0
+	>=dev-dotnet/dbus-sharp-glib-0.5:2.0
 	>=x11-libs/libnotify-0.4.5"
 DEPEND="${RDEPEND}"
 

